package.json

```diff
@@ -34,6 +34,7 @@
     "lint": "npm-run-all --continue-on-error \"lint:*\"",
     "lint:dependency-version-consistency": "check-dependency-version-consistency . --ignore-package=@hashengine/apache-arrow-js-bundle",
     "lint:eslint": "yarn workspaces run lint:eslint",
+    "lint:lockfile-lint": "lockfile-lint --path yarn.lock --allowed-hosts yarn --allowed-schemes \"https:\"",
```

Comment:
Does `lockfile-lint` block non-npm registries by default?

Reply:
Yep, that’s what we want from it. `--allowed-hosts yarn` makes sure we only get packages from `registry.yarnpkg.com`.

Reply:
Alex, do you think it would be preferable to use the exact host we want to match on instead of using the `yarn` shorthand?

I think being clear about the exact host is good for anyone trying to debug why `registry.yarnpkg.com` is what is allowed even though `--allowed-hosts` is '`yarn`'.